integrations/twilio/webhooks/auth.go

```diff
@@ -20,6 +20,7 @@ const (
 )
 
 type Vars struct {
+	AuthType   string
```

Comment:
Note that `EncodeVars` uses the field's name, i.e. `AuthType`, but it should be `authType`. What you did in AWS should work here (although I would argue that might want to remove `EncodeVars` entirely, but that's beyond the scope of this PR).

Reply:
I started removing EncodeVars but it broke a lot of stuff that had nothing to do with this PR, so I left it. Should I open a new PR for that?

Reply:
Done